Animations/CommonOpacityProps.js

```diff
@@ -1,8 +1,8 @@
 var assign = require('object-assign');
```

Comment:
Add 'use strict'; on line one and leave a newline between that and variable declarations.